spec/airport.spec.js

```diff
@@ -0,0 +1,603 @@
+import Airport from '../src/airport.js';
+import { assertEquals, assertFalse, assertThrows, assertTrue } from '../test_framework.js';
+
+console.log('------------------------------');
+console.log('Running tests for airport.js');
+let passCount = 0, failCount = 0;
+
+// Test 1
+{
+	console.log('------------------------------');
+	console.log('Test 1. (land() test). Empty airport, valid plane. => 1')
+
+	let input, expected, actual, result, airport, plane, weather, srandom;
```

Comment:
Irrelevant variable declarations should have been checked for and removed.
- To make sure to review the code thoroughly before creating a PR for the following challenges.